packages/web/public/static/locales/en/comment.json

```diff
@@ -4,7 +4,7 @@
   "save": "Save",
   "cancel": "Cancel",
   "submit": "Submit",
-  "addCommentPlaceholder": "Add a comment...",
+  "addCommentPlaceholder": "Add a Markdown comment...",
```

Comment:
```suggestion
  "addCommentPlaceholder": "We now support Markdown in comments!! Try using **bold** or *italic*!",
```

Reply:
Hmm, this message looks a bit bulky especially with inline comments. I think `Add a Markdown comment...` looks more streamlined but is also not as clear. What do you think?

<img width="495" alt="Screen Shot 2021-08-07 at 10 23 54 PM" src="https://user-images.githubusercontent.com/351124/128618675-5cd15739-a538-480e-8927-48f40e15d5fc.png">